assets/src/edit-story/components/helpCenter/toggle/index.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { BEZIER } from '../../../../animation';
+import {
+  themeHelpers,
+  Icons,
+  NotificationBubble,
```

Comment:
You'll want to import `BUTTON_TYPES, BUTTON_VARIANTS,BUTTON_SIZES, Button` to do what i suggest here. 

variant is default rectangle and size is default medium, so adding those are mostly for readability.